novaboot

```diff
@@ -1150,9 +1159,19 @@ host server {
 
 if (defined $dhcp_tftp || defined $tftp) {
     $tftp_port ||= 69;
+    # Generate TFTP mapfile
+    open(my $fh, '>', "$builddir/mapfile");
+    print $fh "# Some PXE clients (mainly UEFI) have bug. They add zero byte to the end of the
+# path name. This rule removes it
+r     \\.efi.*   \\.efi";
+    close($fh);
     # Unfortunately, tftpd requires root privileges even with
     # non-privileged (>1023) port due to initgroups().
-    system_verbose("sudo in.tftpd --listen --secure -v -v -v --pidfile tftpd.pid  --address :$tftp_port $builddir");
+    system_verbose("sudo in.tftpd --listen --secure -v -v -v --pidfile tftpd.pid -m mapfile --address :$tftp_port $builddir");
+
+    # Prepare a GRUB netboot directory
+    system_verbose("grub-mknetdir --net-directory=$builddir --directory=/usr/lib/grub/x86_64-efi");
+    system_verbose("grub-mknetdir --net-directory=$builddir --directory=/usr/lib/grub/i386-pc");
```

Comment:
Except for this hardcoded path (without checking that it exists and error handling) other changes look good.